packages/flask/src/constants/index.ts

```diff
@@ -52,6 +52,10 @@ export const UIConstant = {
             right: 8,
             bottom: 8,
         },
+        circleScale: {
+            notActive: 1,
+            Active: 1.5,
```

Comment:
Can we use a lowercase here to make the "camelCase" be consistent?
```suggestion
            active: 1.5,
```